src/lib.rs

```diff
@@ -769,6 +773,13 @@ impl Config {
         self.dgram_recv_max_queue_len = recv_queue_len;
         self.dgram_send_max_queue_len = send_queue_len;
     }
+
+    /// Sets the maximum datagram size used by the congestion control.
+    ///
+    /// The default and minimum value is `1200`.
+    pub fn set_max_datagram_size(&mut self, v: usize) {
+        self.max_datagram_size = cmp::min(v, recovery::MAX_DATAGRAM_SIZE);
```

Comment:
I don't think this does what you intend? If the specification minimum is 1200 then don't we want something like this?

```
self.max_datagram_size = cmp::max(v, 1200);
``

Reply:
Fixed. Thanks